doc/modelling/binding/index.rst

```diff
@@ -46,6 +46,29 @@ This can be achieved by a (nonlinear) parameter transform
         F\left( k_{\text{eq},i}, k_{d,i} \right) = \begin{pmatrix} k_{\text{eq},i} k_{d,i} \\ k_{d,i} \end{pmatrix} \text{ with Jacobian } J_F\left( k_{\text{eq},i}, k_{d,i} \right) = \begin{pmatrix} k_{d,i} & k_{\text{eq},i} \\ 0 & 1 \end{pmatrix}.
     \end{aligned}
 
+
+.. _ldf_model:
+
+Linear Driving Force (LDF)
+---------------------------
+
+A quantitative description of an actual separation process requires the simultaneous solution of the diffusion equation and the bulk material balances, for which no analytical solutions exist. It is therefore computationally convenient to approximate adsorption rate by a time-independent expression given as:
+
+.. math::
+    \begin{aligned}
+        \frac{\mathrm{d} q_i}{\mathrm{d} t} = k_{kkin_{,i}}\left(q^*_i - q_i\right) && i = 0, \dots, N_{\text{comp}} - 1.
```

Comment:
I think there is one `k` too many in `k_{kkin_{,i}}`. Alternatively we could use `k_{ldf_{,i}}` since the `k` itself already implies that it's a kinetic coefficient.

Also, I'm not sure if the `i` needs to be in a lower subscript level. Then it would be 
` k_{kin,i}` or ` k_{ldf,i}`

Reply:
True, it is one 'k' too many. Will correct it asap.
I think `k_{kin,i}` makes more sense in this regard and I will fix the equation accordingly.
Thanks for your review.